pydocmd/__main__.py

```diff
@@ -39,244 +39,254 @@
 
```

Comment:
Could you please revert these indentation changes as well?

Reply:
Done